src/common/OptionsMenu/OptionsMenu.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { CSSTransition } from 'react-transition-group'
+
+import PopUpDialog from '../PopUpDialog/PopUpDialog'
+
+import './optionsMenu.scss'
+
+const OptionsMenu = React.forwardRef(({ children, show, timeout }, ref) => {
+  return (
+    <CSSTransition
+      in={show}
+      timeout={timeout}
+      classNames="options_menu__transition"
```

Comment:
"_" is used for the modifier (example: `menu_disabled`, `menu_valid`, etc.)
In this case you can change the class to `options-menu-transition`